XamarinCommunityToolkit/Microsoft.Toolkit.Xamarin.Forms.csproj

```diff
@@ -112,4 +111,7 @@
       <Compile Include="**\*.tizen.cs" />
       <Compile Include="**\*.tizen.*.cs" />
     </ItemGroup>
+    <ItemGroup>
+      <Compile Include="Behaviors\MaskedBehavior.cs" />
```

Comment:
MaskedBehavior.shared.cs

You should create this file in the file system and put it into the proper folder. This csproj shouldn't be modified

Reply:
@AndreiMisiukevich what would be the benefit of making it bindable?

Reply:
Ability to control it from VM via bindings

Reply:
this file must be unchanged